lib/tests/basic-call.rb

```diff
@@ -87,6 +92,42 @@
 
 end
 
+TestDefinition.new("Basic Call - Tel URIs") do |t|
+  caller = t.add_endpoint
+  callee = t.add_endpoint
+
+  ringing_barrier = Barrier.new(2)
+
+  t.add_quaff_setup do
+    caller.register
+    callee.register
+  end
+
+  t.add_quaff_scenario do
+    tel = sip_to_tel(callee.uri)
+    call = caller.outgoing_call(tel)
+
+    call.send_request("MESSAGE", "hello world\r\n",
```

Comment:
All I care about here is whether the request is routed from A to B - non-INVITE transactions are simpler (no 100, no ACK) and so I prefer them where possible to reduce boilerplate.